core/src/main/java/io/questdb/log/LogRollingFileWriter.java

```diff
@@ -108,6 +119,25 @@ public void bindProperties(LogFactory factory) {
             nRollSize = Long.MAX_VALUE;
         }
 
+        if (this.sizeLimit != null) {
+            try {
+                nSizeLimit = Numbers.parseLongSize(this.sizeLimit);
+                if(nSizeLimit < nRollSize) {
```

Comment:
Still not properly formatted:
```suggestion
                if (nSizeLimit < nRollSize) {
```